packages/shared/package.json

```diff
@@ -21,7 +21,7 @@
     "graphql": "^15.5.0",
     "graphql-request": "^3.4.0",
     "idb-keyval": "^5.0.6",
-    "next": "^12.2.2",
+    "next": "14.2.8",
```

Comment:
Probably not just next but main bundle jumped by ~19kb https://vercel.com/dailydotdev/daily-webapp/8bvbNVp4a3bn1sG6wYfkv5W6nKpr#L157 vs main, something to maybe take a look at

Reply:
https://github.com/vercel/next.js/issues/58047